package.json

```diff
@@ -22,6 +22,6 @@
     "@types/react": "17.0.33",
     "eslint": "7.32.0",
     "eslint-config-next": "12.0.9",
-    "typescript": "4.4.4"
+    "typescript": "4.5.5"
```

Comment:
```suggestion
    "typescript": "4.2.0"
```

Reply:
Any reason why you need a downgraded TS version?

Reply:
memes

Reply:
and dreams

Reply:
Ahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh of course my mistake

Reply:
Typescript 4.2.0 >> any version